continuous_integration/gpuci/environment.yaml

```diff
@@ -28,7 +28,6 @@ dependencies:
 - pytest-xdist
 - pytest
 - python=3.9
-- rust=1.62.1
```

Comment:
Switching over to `rustup` in gpuCI with https://github.com/rapidsai/dask-build-environment/pull/48